src/pemfile.rs

```diff
@@ -1,59 +1,17 @@
-use alloc::borrow::ToOwned;
 use alloc::format;
 use alloc::string::String;
 use alloc::vec::Vec;
 #[cfg(feature = "std")]
 use core::iter;
-use core::ops::ControlFlow;
 #[cfg(feature = "std")]
 use std::io::{self, ErrorKind};
 
-use pki_types::{
-    CertificateDer, CertificateRevocationListDer, CertificateSigningRequestDer, PrivatePkcs1KeyDer,
-    PrivatePkcs8KeyDer, PrivateSec1KeyDer, SubjectPublicKeyInfoDer,
-};
-
-/// The contents of a single recognised block in a PEM file.
-#[non_exhaustive]
-#[derive(Debug, PartialEq)]
-pub enum Item {
-    /// A DER-encoded x509 certificate.
-    ///
-    /// Appears as "CERTIFICATE" in PEM files.
-    X509Certificate(CertificateDer<'static>),
-
-    /// A DER-encoded Subject Public Key Info; as specified in RFC 7468.
-    ///
-    /// Appears as "PUBLIC KEY" in PEM files.
-    SubjectPublicKeyInfo(SubjectPublicKeyInfoDer<'static>),
-
-    /// A DER-encoded plaintext RSA private key; as specified in PKCS #1/RFC 3447
-    ///
-    /// Appears as "RSA PRIVATE KEY" in PEM files.
-    Pkcs1Key(PrivatePkcs1KeyDer<'static>),
-
-    /// A DER-encoded plaintext private key; as specified in PKCS #8/RFC 5958
-    ///
-    /// Appears as "PRIVATE KEY" in PEM files.
-    Pkcs8Key(PrivatePkcs8KeyDer<'static>),
-
-    /// A Sec1-encoded plaintext private key; as specified in RFC 5915
-    ///
-    /// Appears as "EC PRIVATE KEY" in PEM files.
-    Sec1Key(PrivateSec1KeyDer<'static>),
-
-    /// A Certificate Revocation List; as specified in RFC 5280
-    ///
-    /// Appears as "X509 CRL" in PEM files.
-    Crl(CertificateRevocationListDer<'static>),
-
-    /// A Certificate Signing Request; as specified in RFC 2986
-    ///
-    /// Appears as "CERTIFICATE REQUEST" in PEM files.
-    Csr(CertificateSigningRequestDer<'static>),
-}
+use pki_types::pem as parent;
+pub use pki_types::pem::Item;
 
 /// Errors that may arise when parsing the contents of a PEM file
+///
+/// This differs from pki-types because it is `PartialEq`
```

Comment:
Seems like we might as well use a rustdoc-link here and say  "This differs from [`pki-types::pem::Error`] ..". WDYT?